package.json

```
@@ -4,14 +4,15 @@
   "scripts": {
     "build": "lerna run build",
     "watch": "lerna run --parallel watch",
-		"lint": "eslint -c .eslintrc.js \"{lean4-infoview-api,lean4-infoview,vscode-lean4}/src/**/*.{ts,tsx}\""
+    "watchTest": "lerna run --parallel watchTest",
```

```suggestion
    "watchTest": "lerna run --parallel watchTest",
    "test": "lerna run test",
```
So that just `npm run test` at the top-level works.

---

Ah, got it, thanks